mypy/stubtest.py

```diff
@@ -269,11 +269,13 @@ def verify_typeinfo(
             mangled_entry = "_{}{}".format(stub.name, entry)
         stub_to_verify = next((t.names[entry].node for t in stub.mro if entry in t.names), MISSING)
         assert stub_to_verify is not None
-        yield from verify(
-            stub_to_verify,
-            getattr(runtime, mangled_entry, MISSING),
-            object_path + [entry],
-        )
+        try:
+            runtime_attr = getattr(runtime, mangled_entry)
+        except Exception:
```

Comment:
Do we need to log it somehow?

Reply:
Not sure there's a good framework for that. Perhaps we can add a constant similar to MISSING, or fall back to a runtime value of Any if a non-AttriibuteError exception is thrown.